modules/nf-core/diamond/blastp/meta.yml

```diff
@@ -22,11 +22,11 @@ input:
   - fasta:
       type: file
       description: Input fasta file containing query sequences
-      pattern: "*.{fa,fasta}"
+      pattern: "*.{fa,fasta,fa.gz,fasta.gz}"
   - db:
-      type: directory
-      description: Directory containing the protein blast database
-      pattern: "*"
+      type: file
+      description: File of the indexed DIAMOND database
+      pattern: "*.{dmnd}"
```

Comment:
Don't need the curly brackets here, not a list

Reply:
Done